spec/active_type/record_extension_spec.rb

```diff
@@ -53,6 +56,9 @@ class ExtendedRecordWithValidations < ExtendedActiveTypeRecord
   it 'has the same model name as the base class' do
     expect(subject.class.model_name.singular).to eq(RecordExtensionSpec::Record.model_name.singular)
   end
+  it 'has the same STI type as the base class' do
```

Comment:
I feel like the description here is not very helpful. Perhaps something like `it does not require an STI type condition`.